lib/codemirror.css

```diff
@@ -270,6 +270,7 @@ div.CodeMirror span.CodeMirror-nonmatchingbracket {color: #f22;}
 .CodeMirror-widget {}
 
 .CodeMirror-rtl pre { direction: rtl; }
+.CodeMirror-ltr pre { direction: ltr; }
```

Comment:
I think you can just style `.CodeMirror pre`, _before_ the other rule, and don't need an additional class. Would that work? I'd prefer not to have another always-present class on the main container.